### DISCUSSION TITLE
getting error while running the code

### DISCUSSION THREAD

**Discussion by bhishmakajaria** [devlikeapro/waha#168] [Q&A]
![WhatsApp Image 2023-08-13 at 23 46 08](https://github.com/devlikeapro/whatsapp-http-api/assets/67088711/c77af488-91da-4e75-9fc3-add61df8c2cb)

We are trying to run the code locally on windows os but we are getting the above error. can you please help

**Comment by allburov** [comment]
you should change the path according to your environment - https://github.com/devlikeapro/whatsapp-http-api/blob/5a628907edd3d67201257a256c73c3da8443813f/src/core/abc/session.abc.ts#L44C1-L52

I think the right way would be to setup path to chrome via environment variable `WAHA_CHROME_PATH`, but Windows is not our target platform, so feel free to create a PR for that, I'll be happy to help with that!

**Comment by bhishmakajaria** [comment]
Dear Devlikeapro/Whatsapp-Http-Api,

facing the below issue when creating docker image. Let me know how we can
resolve this issue as all npm seems deprecated and *whiskeysockets/baileys *git
is not fetching the files.

Please help soon.

[image: image.png]

**Reply by allburov** [reply]
Today we released a new version using that dockerfile, no issues. 
Please make sure you're using the latest dockerfile and source code
